### PR TITLE
adding todo

### PR DESCRIPTION
**Pull request by livneam**
# My Very Cool Editing...
- this is my edit... test

meme

### CODE REVIEW

**Comment by livneam** [comment]
dealt with few aspects.
1. register + login (include new screen post login view which display top 5 games and last month )
2. inventory 
3. clients management
4. category

File: app/app.js [hunk 1]
```diff
@@ -51,10 +56,15 @@ angular
                 controllerAs: 'catalog'
             })
             .when('/clientsManagement', {
-                templateUrl: 'components/clientsManagement/clientsManagement.html',
+                templateUrl: 'components/clientsManagement/clientsmanagement.html',
```
Comment (eliranach) [inline app/app.js]: please keep camelCase!!! 👎 change the file to `clientsManagement.html` next time!

😢

File: app/components/about/about.html [hunk 2]
```diff
@@ -1 +1,20 @@
...
+  <h3>Eliran's website</h3>
+  <span><a href="https://nachmani-eliran.000webhostapp.com" target="_blank">Eliran's website</a></span>
+  <br>
+  <br>
+  <h3>Amit's website</h3>
+  <span><a href="http://livneam.site88.net/" target="_blank">Amit's website</a></span>
+  <br>
+  <br>
+  <h3>Previous work together - Pacman game</h3>
+  <span><a href="http://pacman-amit-eliran.000webhostapp.com/" target="_blank">Pacman game</a></span>
```
Comment (eliranach) [inline app/components/about/about.html:20]: 👍 very nice!
borat

File: app/components/catalog/catalog.html [hunk 3]
```diff
@@ -1 +1,31 @@
...
+<div class="content">
+    <ul id="game-list">
+      <li  ng-repeat="rec in catalog.Recgames">
+        <img class="details" ng-src="{{catalog.baseUrl + rec.picPath}}" style="margin: -12px 10px 0 0; float:left ;width:50px;"
+             ng-click="gameDetails(rec)"/>
+        <h3>{{rec.gameName}}, By:{{rec.publisher}}</h3>
+        <span class="price">{{rec.price | currency}}</span>
+      </li>
+    </ul>
+</div>
```
Comment (eliranach) [inline app/components/catalog/catalog.html:31]: 👍

File: app/components/catalog/catalog.js [hunk 4]
```diff
@@ -8,10 +8,50 @@
...
+                products.getLastMonthItems()                
+                .then(function(res)
+                {
+                  $log.info("getLastMonthItems response:", res);
+                  catalog.MonthGames = res;                  
+                });
+
+            });
+  
+      });
```
Comment (eliranach) [inline app/components/catalog/catalog.js:48]: 👍

File: app/components/clientsManagement/clientsmanagement.js [hunk 5]
```diff
@@ -8,10 +8,40 @@
  * Controller of the gameStoreApp
  */
 angular.module('gameStoreApp')
-  .controller('ClientsmanagementCtrl', function () {
+  .controller('ClientsManagementCtrl', function (commonHttp, clients,$log,$location) {
```
Comment (eliranach) [inline app/components/clientsManagement/clientsmanagement.js:11]: 👍 good naming convention - using camelCase

File: app/components/main/main.js [hunk 6]
```diff
@@ -47,7 +47,10 @@ angular.module('gameStoreApp')
         {
           label: 'Catalog',
           route: '#/catalog',
-          roles: ['guest', 'user', 'admin']
+          roles: ['guest', 'user', 'admin'],
+          controller: 'CatalogCtrl'
```
Comment (eliranach) [inline app/components/main/main.js]: # warning:
probably we will have difficalties here with my code at this file!

File: app/index.html [hunk 7]
```diff
@@ -46,13 +46,13 @@
                             <a href="{{::item.route}}">{{item.label}}</a>
                         </li>
                     </ul>
-
+                    {{user.role}}
```
Comment (eliranach) [inline app/index.html]: # WHY??!?!?
❓ @livneam

Comment (livneam) [reply app/index.html]: deleted ...  just wanted to check if the role is fetchable

File: app/components/main/main.js (hunk 6 above)
Comment (livneam) [reply app/components/main/main.js]: clear.  deleted those,

File: app/app.js [hunk 8]
```diff
@@ -56,7 +56,7 @@ angular
                 controllerAs: 'catalog'
             })
             .when('/clientsManagement', {
-                templateUrl: 'components/clientsManagement/clientsmanagement.html',
+                templateUrl: 'components/clientsManagement/clientsManagement.html',
```
Comment (eliranach) [inline app/app.js]: 👍

**Review by eliranach — APPROVED** [review]

**Comment by livneam** [comment]
Yei!

**Pull request merged**